src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.ts

```diff
@@ -82,13 +85,15 @@ export class AnnotationContextMenuComponent implements OnInit, OnDestroy {
 				this.hoverFeatureId = hoverFeatureId;
 			}),
 			this.annotations.events.onSelect.subscribe((selected: string[]) => {
-					this.selection = selected;
-					this.selectedTab = this.selection.reduce((prev, id) => ({
-						...prev,
-						[id]: this.selectedTab[id]
-					}), {});
+				if (selected.length > 0) {
+					this.communicator.logMessages.emit(`Opening annotations context menu`);
```

Comment:
maybe create an object or a specific config for the logs from the communicator

Reply:
What do you mean?
What for?

Reply:
there are different messages specifically for the communicator object.
i suggested that we define a const in the communicator class that includes all of the messages related to it.
for example:
this.communicator.logMessages.emit(this.communicator.logs.openContextMenu);

Reply:
Sounds good to me. I will implement this after resolving the conflicts.

Reply:
great!